packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -107,10 +104,11 @@ class ComponentWrapper extends React.Component<IComponentWrapperProps<IDevSiteDe
             ? `${this.props.managedClasses.componentWrapper__transparent} ${classNames}`
             : classNames;
 
-        return this.props.active && this.props.view === ComponentViewTypes.examples
-            ? `${classNames} ${this.props.managedClasses.componentWrapperExamples} ${this.props.managedClasses.componentWrapper__active}`
-            : this.props.view === ComponentViewTypes.detail
-            ? `${classNames} ${this.props.managedClasses.componentWrapper__active}`
+        classNames = this.props.active
+            ? `${this.props.managedClasses.componentWrapper} ${this.props.managedClasses.componentWrapper__active}`
+            : this.props.managedClasses.componentWrapper;
```

Comment:
I think @chrisdholt previous comment still applies here, we are going to have a situation where if the component is active and transparent you will be overwriting and not including the transparent class. You can just replace `this.props.managedClasses.componentWrapper` on 108 and 109 with `classNames`.